packages/components/src/components/SquiggleViewer/ValueWithContextViewer.tsx

```diff
@@ -239,10 +257,19 @@ export const ValueWithContextViewer: FC<Props> = ({
       <div ref={ref}>
         {header !== "hide" && (
           <header
+            tabIndex={viewerType === "tooltip" ? undefined : 0}
             className={clsx(
-              "flex justify-between group pr-0.5",
-              header === "large" ? "mb-2" : "hover:bg-stone-100 rounded-sm"
+              "flex justify-between group pr-0.5 hover:bg-stone-100 rounded-sm focus-visible:outline-none",
+              focused
```

Comment:
Should this be `isFocused`?

Otherwise it's applied when anything is focused, and lists in focused mode look bad:

<img width="433" alt="Screenshot 2024-01-24 at 14 15 11" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/5256237a-e0d8-4a32-95b2-04f5b595d0a6">

Reply:
Good catch! looks like a mistake